FirstProject/new-evaluation.html

```diff
@@ -482,10 +474,10 @@ <h2 class="description_section" style="color: #03A9F4;">Workflow, Leadership & S
             </fieldset>
         </form>
 
-        <form class="language_form" style="text-align:center;">
-            <fieldset class="field_section" style="display:inline; width: 61%;">
+        <form class="language_form">
+            <fieldset class="field_section">
                 <legend><h2>NodeJS</h2></legend>
```

Comment:
Because I want my title to be an H2, to avoid the font-size styling.

It is a bad practice?